src/de/grobox/liberario/fragments/DirectionsFragment.java

```diff
@@ -390,7 +391,7 @@ private void search() {
 				pd.setCancelable(false);
 				pd.setIndeterminate(true);
 				pd.setButton(DialogInterface.BUTTON_NEGATIVE, "Cancel", new DialogInterface.OnClickListener() {
-					             @Override
+ 					             @Override
```

Comment:
Can't make out the difference here. Looks like a space. But i can't figure it out in Android Studio.